paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -163,25 +163,31 @@ def mark_for_deployment(git_url, deploy_group, service, commit):
         sha=commit,
     )
 
-    try:
-        remote_git.create_remote_refs(git_url=git_url, ref_mutator=ref_mutator, force=True)
-    except Exception as e:
-        loglines = ["Failed to mark %s for deployment in deploy group %s!" % (commit, deploy_group)]
-        for line in str(e).split('\n'):
-            loglines.append(line)
-        return_code = 1
-    else:
-        loglines = ["Marked %s for deployment in deploy group %s" % (commit, deploy_group)]
-        return_code = 0
+    max_attempts = 3
+    for attempt in range(1, max_attempts + 1):
```

Comment:
Maybe add a small sleep just to increase the odds the retries work?

Reply:
added